### PR TITLE
Created models

### PR DESCRIPTION
**Pull request by MifRea1**

### CODE REVIEW

File: test/factories/tasks.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+FactoryBot.define do
+  factory :task do
+    name { "MyString" }
+    description { "MyText" }
+    author_id { 1 }
+    assignee_id { 1 }
+    state { "MyString" }
```
Comment (artempartos) [inline test/factories/tasks.rb:7]: configure factories through sequnces

File: test/models/admin_test.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class AdminTest < ActiveSupport::TestCase
```
Comment (artempartos) [inline test/models/admin_test.rb]: remove useless model tests

Comment (MifRea1) [reply test/models/admin_test.rb]: done

**Comment by artempartos** [comment]
👍

**Pull request merged**